src/course_bot/presentation.clj

```diff
@@ -115,8 +115,8 @@
     :drop-state-1 "Мы сбрасываем ваше состояние на %s"
     :descriptions "%s описаний"
     :all-scheduled-description-by-group "Файл со всеми запланированными описаниями по группам:"
-    :yes "да"
-    :no "нет"
+    :yes "yes"
+    :no "no"
```

Comment:
We don't need to use translation here.

Reply:
I'm trying to fix it this way: 
![image](https://github.com/ryukzak/course-bot/assets/91891085/a257c94e-3c3b-46f0-a9b0-3ae4273793ec)

The logic in the general.clj file is implemented in a similar way, but unfortunately this error crashes: 
![image](https://github.com/ryukzak/course-bot/assets/91891085/6f37477e-aed1-4296-a78c-f60923d63dab)

I'm trying for the second day to figure out how to fix it, but I didn't come to anything. Do you know how to fix it?